docs/examples/getting_started.md

```diff
@@ -98,30 +114,28 @@ Optional Arguments
 Follow [here](./../cmf_client/cmf_client.md#cmf-init) for more details.
 
 ## Project execution
-To execute the example pipeline, run the 
-[test_script.sh](../../examples/example-get-started/test_script.sh) 
-file (before that, study the contents of that file). Basically, that script will run a sequence of steps
-common for a typical machine learning project - getting raw data, converting it into machine learning train/test splits,
-training and testing a model. The execution of these steps (and parent pipeline) will be recorded by the CMF.
+To execute the example pipeline, run the
+[test_script.sh](../../examples/example-get-started/test_script.sh) file. Basically, that script will run the sequence of steps
+described in the Overview section above.
 ```shell
 # Run the example pipeline
 sh ./test_script.sh
 ```
 
-This script will run the pipeline and will store its metadata in a sqlite file named mlmd. Verify that all stages are 
-done using `git log` command. You should see commits corresponding to the artifacts that were created.
+This script will run the pipeline and will store its metadata in a sqlite file named mlmd. Verify that all were completed by running the `git log` command. Each created artifact should be associated with its own commit.
 
-Under normal conditions, the next steps would be to: (1) execute the `cmf artifact push` command to push the artifacts to the central artifact repository and (2) execute the `cmf metadata push` command to track the metadata of the generated artifacts on a common [cmf server](./../cmf_server/cmf-server.md).
+In typical produciton environments, the next steps would be to: (1) execute the `cmf artifact push` command to push the artifacts to the central artifact repository and (2) execute the `cmf metadata push` command to integrate the metadata of the generated artifacts on a common [cmf server](./../cmf_server/cmf-server.md).
```

Comment:
we no longer add a commit for every created artifact as it was making too many git calls. We now do only a single commit at the start and end of execution. we do git add when a dataset is logged but do not commit it then. we commit at the end